packages/react-storage/src/components/StorageBrowser/adapters/createAmplifyAuthAdapter/createAmplifyListLocationsHandler.ts

```diff
@@ -25,11 +20,11 @@ export const createAmplifyListLocationsHandler = (): ListLocations => {
     const { locations }: { locations: ListPathsOutput['locations'] } =
       await listPaths();
 
-    const sanitizedLocations: LocationAccess[] = locations.map(
+    const sanitizedLocations = locations.map(
       ({ bucket, permission, prefix, type }) => {
         return {
           type,
-          permission: mapAmplifyPermissions(permission),
+          permissions: parseAmplifyAuthPermission(permission),
           scope: `s3://${bucket}/${prefix}`,
```

Comment:
AS part of this work, we also wanted to split `scope` into `bucket` & `prefix` for both managed Auth and Amplify Auth and make ListLocations output to match `LocationData`. This will eliminate the need for separate ListPath types. 

Can do as followup. But do we still wanna do this? cc: @calebpollman

Reply:
Yes, picking that up